framework/doc/content/source/systems/LinearSystem.md

```diff
@@ -0,0 +1,108 @@
+# LinearSystem
+
+## Description of a linear system in MOOSE
+
+In MOOSE, a `LinearSystem` denotes a linear algebraic equation system that
+arises from the spatial discretization of a linearized partial differential equation.
+Unlike [NonlinearSystem.md], where Newton's method is used, with a `LinearSystem` the user can
```

Comment:
```suggestion
Unlike [NonlinearSystem.md], where a nonlinear solver method based on Newton's method is used, with a `LinearSystem` the user can
```

if there is a better way to say Newton + JFNK + PJFNK use that

Reply:
All of them are Newton's method. The only difference is the way they preconditions and/or approximate the Jacobian.

Reply:
I added some clarification.